app/src/main/java/org/oppia/app/home/HomeActivityController.kt

```diff
@@ -12,7 +13,7 @@ class HomeActivityController @Inject constructor(private val activity: AppCompat
     activity.setContentView(R.layout.home_activity)
     activity.supportFragmentManager.beginTransaction().add(
       R.id.home_fragment_placeholder,
-      HomeFragment()
+      AudioFragment()
```

Comment:
Let's not keep this since it's breaking the home screen flow.

Reply:
Done. changes to `HomeFragment()`